psi4/src/export_fock.cc

```diff
@@ -155,8 +157,7 @@ void export_fock(py::module &m) {
         .def("add_space", &DFHelper::add_space)
         .def("initialize", &DFHelper::initialize)
         .def("print_header", &DFHelper::print_header)
-        .def("add_transformation", &DFHelper::add_transformation, py::arg("name"), py::arg("key1"), py::arg("key2"),
-             py::arg("order") = "Qpq")
+        .def("add_transformation", &DFHelper::add_transformation, "name"_a, "key1"_a, "key2"_a, "order"_a = "Qpq")
```

Comment:
Having done some I'm-really-trying-to-add-a-feature-but-ended-up-modernizing-cleaning-formatting-fixing-along-the-way projects of late, I agree with @fevangelista that it's a good idea to `git reset` and do a PR of the uncontroversial changes like `py::arg` --> `_a` and de-c++11 so that those go in sooner and the foundations for your bigger changes aren't shifting.